### PR TITLE
Navi button

### PR DESCRIPTION
**Pull request by jkusa**
## Description

Refactor buttons with NaviButton components

Needs Ember 3.4 or https://github.com/rwjblue/ember-angle-bracket-invocation-polyfill

## Proposed Changes

- Add NaviButton component
- Add NaviButton to dummy app
- Use NaviButton in reports footer

## License

I confirm that this contribution is made under an MIT license and that I have the authority necessary to make this contribution on behalf of its copyright owner.

### CODE REVIEW

File: packages/core/addon/components/navi-button.js [hunk 1]
```diff
@@ -0,0 +1,28 @@
+/**
+ * Copyright 2019, Yahoo Holdings Inc.
+ * Licensed under the terms of the MIT license. See accompanying LICENSE.md file for terms.
+ *
+ * Usage:
+ *   <NaviButton
+ *     @type="seconary"
```
Comment (mattgraham1995) [inline packages/core/addon/components/navi-button.js]: typo

File: packages/core/addon/components/navi-button.js [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+import Component from '@ember/component';
+import { computed } from '@ember/object';
+import layout from '../templates/components/navi-button';
+
+export default Component.extend({
+  layout,
+  tagName: '',
+  class: computed('disabled', 'type', function() {
+    const type = this.type || 'primary';
+    const isDisabled = this.disabled;
```
Comment (mattgraham1995) [inline packages/core/addon/components/navi-button.js]: Does this need to be assigned if this const is only used once?

File: packages/core/app/styles/navi-core/components/navi-button.less [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+  transition: 300ms;
+
+  &--primary {
+    background: linear-gradient(to right, @denali-brand-600, @denali-brand-700);
+    color: @navi-white;
+    min-width: 120px;
+
+    &:hover {
+      background: linear-gradient(to right, @denali-brand-700, @denali-brand-800);
+      background: linear-gradient(to right, #3570f4, #0044e0);
```
Comment (mattgraham1995) [inline packages/core/app/styles/navi-core/components/navi-button.less]: two background properties?

Comment (jkusa) [reply packages/core/app/styles/navi-core/components/navi-button.less]: oops

**Pull request merged**